mne/chpi.py

```diff
@@ -149,6 +150,115 @@ def head_pos_to_trans_rot_t(quats):
     return translation, rotation, t
 
 
+def _apply_quat(quat, pts, move=True):
+    """Apply MaxFilter-formatted head position parameters to points."""
+    trans = np.concatenate(
+        (quat_to_rot(quat[:3]),
+         quat[3:][:, np.newaxis]), axis=1)
+
+    return(apply_trans(trans, pts, move=move))
+
+
+def extract_head_pos_ctf(raw, gof_limit=0.98):
+    """Extract MaxFilter-formatted head position parameters from ctf dataset.
```

Comment:
I was just following the pattern from the other methods.

I do find this pos/quat nomenclature a bit confusing.

For instance the method `_apply_quat` should really be in mne.transforms but the `quat` isn't quite a quaternion so I left it in chpi.
I guess I could refactor it into taking a real quaternion and a translation and then put it in mne.transforms ,what do you think?